src/main/kotlin/app/revanced/patches/all/misc/packagename/ChangePackageNamePatch.kt

```diff
@@ -1,16 +1,26 @@
-package app.revanced.patches.all.misc.packagename.patch
+package app.revanced.patches.all.misc.packagename
 
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.data.ResourceContext
-import app.revanced.patcher.patch.*
-import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.patch.PatchException
+import app.revanced.patcher.patch.ResourcePatch
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patcher.patch.options.types.StringPatchOption.Companion.stringPatchOption
 import org.w3c.dom.Element
 
-@Patch(false)
-@Name("Change package name")
-@Description("Changes the package name. Appends \".revanced\" to the package name by default.")
-class ChangePackageNamePatch : ResourcePatch {
+@Patch(
+    name = "Change package name",
+    description = "Changes the package name. Appends \".revanced\" to the package name by default.",
```

Comment:
❓ : Redundant wording
> The patch's name: `Change package name`, the description include `Changes the package name.` which is unnecessary and should be remove.

```suggestion
    description = "Appends \".revanced\" to the package name by default.",
```